.travis.yml

```diff
@@ -1,13 +1,13 @@
 # Travis CI (MIT License) configuration file
 # @link https://travis-ci.org/
 
-# Use new container based environment
-sudo: false
-
 # Declare project language.
 # @link http://about.travis-ci.org/docs/user/languages/php/
 language: php
 
+# Use new container based environment
+sudo: false
```

Comment:
Why do it later? Logically we would want to know where we're going to run stuff, before knowing what we're going to run?